wit/bindgen/generator.go

```diff
@@ -679,13 +679,13 @@ func (g *generator) recordRep(file *gen.File, dir wit.Direction, r *wit.Record,
 	exported := len(goName) == 0 || token.IsExported(goName)
 	var b strings.Builder
 	b.WriteString("struct {\n")
-	stringio.Write(&b, "_ ", file.Import(g.opts.cmPackage), ".HostLayout")
+	stringio.Write(&b, "_ ", file.Import(g.opts.cmPackage), ".HostLayout", "`json:\"-\"`")
```

Comment:
Don't forget the space before the backtick, or just have a single string with HostLayout and the tag.